tests/network-tests/src/flows/forum/posts.ts

```diff
@@ -144,6 +161,21 @@ export default async function threads({ api, query }: FlowProps): Promise<void>
   // Run compound query node checks
   await Promise.all([updatePostsTextRunner.runQueryNodeChecks(), deletePostsRunner.runQueryNodeChecks()])
 
+  // Moderate removed posts
+  const postModerations = [
```

Comment:
I think it would be useful to have other test cases here like:
- Trying to moderate as worker w/o category permissions
- Trying to moderate as non-forum worker
- Optionally: Trying to moderate as worker with permissions only to `category.parent.parent`

Reply:
You're right, I will try to quickly add some of these tomorrow